source/interface/transport_interface.h

```diff
@@ -1,5 +1,5 @@
 /*
- * coreHTTP v3.0.0
+ * coreHTTP v3.1.0
  * Copyright (C) 2022 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (C) 2024 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
```
Is this missed or intended?

Reply:
There is SSOT check if this header file is the same as the [transport_interface.h](https://github.com/FreeRTOS/coreMQTT/blob/main/source/interface/transport_interface.h) in coreMQTT.
Therefore, we can't just update this file here.